manifests/repo.pp

```diff
@@ -0,0 +1,39 @@
+# == Class: gluster::repo
+#
+# Manages the repo to install gluster from. Currently only Ubuntu is supported.
+#
+# === Parameters
+#
+# [*manage*]
+#   Whether to manage the repository.
+#
+# [*source*]
+#   Repo source. Valid sources are:
+#   * `gluster/glusterfs-3.7`
+#
+class gluster::repo(
+  $manage = true,
+  $source = 'gluster/glusterfs-3.7',
+) {
+  if $manage {
+    case $::osfamily {
+      'Debian': {
```

Comment:
Can we filter on Ubuntu somehow? I know we're not supporting Debian but the PPA won't work on Debian.